singlestoredb/management/workspace.py

```diff
@@ -84,324 +90,7 @@ def get_workspace(
     raise RuntimeError('no workspace group specified')
 
 
-class StageObject(object):
```

Comment:
I'm wondering if we should add a `StageObject = FilesObject` at the end of this file in case people have existing code with type hints that include `StageObject` in them. Otherwise, this is a breaking API change.

Reply:
Thank you for bringing that up! I agree that we should create an alias to ensure backward compatibility.